unity/README.md

```diff
@@ -59,22 +59,24 @@ It is possible to provide a space separated list of desired targets to build and
 > .yamato\scripts\build_yamato.cmd --build NullGC EmbeddingHost --test EmbeddingManaged
 ```
 
-### Using a debug build of the CoreCLR runtime in Unity
+### Using a debug or custom build of the CoreCLR runtime in Unity
 
-Once the CoreCLR runtime has been built locally you can find the runtime artifacts the Unity player needs in `artifacts\bin\microsoft.netcore.app.runtime.win-x64\Debug\runtimes\win-x64`. 
+Once the CoreCLR runtime has been built locally you can find the runtime artifacts Unity needs in one of:
+* `artifacts\bin\microsoft.netcore.app.runtime.win-x64\Debug\runtimes\win-x64`
+* `artifacts/bin/microsoft.netcore.app.runtime.osx-arm64/Debug/runtimes/osx-arm64`
+* replace `Debug` with `Release` if a release build
+
+You can use the script to deploy an update of CoreCLR to your Unity build using:
 
-If you would like to patch a built player with your local build you can use
 ```
-yamato\scripts\build_yamato.cmd --deploy-to-player --config Debug <Unity player build directory>
+.yamato/scripts/build_yamato.sh --config Debug --deploy-to-player ..../build/MacEditor/arm64/Debug/Unity.app/Contents/CoreCLR
```

Comment:
```suggestion
.yamato/scripts/build_yamato.sh --config Debug --deploy-to-player <path-to-coreclr-folder>
```

Reply:
Hmm, how attached are you to this? I don't think it hurts to have the actual unity structure in here; it doesn't leak anything, and it will help Unity devs (who are the main ones that will be reading this).

Reply:
Oh I see below! Yes, great.